ansible/roles/prosody/files/mod_muc_webhooks.lua

```diff
@@ -790,6 +791,52 @@ local function occupant_affiliation_changed(event)
     end
 end
 
+local function handle_transcription_chunk(event)
+    local subject = event.stanza:get_child("subject");
+    if subject then
+        return;
+    end
+
+    if event.stanza.attr.type == "groupchat" then
+        local body = event.stanza:get_child("body")
+        if body then
+            return;
+        end
+
+        local transcription = util.get_final_transcription(event)
+        if transcription then
+            local participant = {
+                ["name"] = transcription["participant"]["name"],
+                ["id"] = transcription["participant"]["identity_id"],
+                ["avatarUrl"] = transcription["participant"]["avatar_url"],
+                ["email"] = transcription["participant"]["email"]
+            }
+            local data = {
+                ["messageID"] = transcription["message_id"],
+                ["participant"] = participant,
+                ["language"] = transcription["language"],
+                ["final"] = transcription["transcript"][util.FIRST_TRANSCRIPT_MESSAGE_POS]["text"]
+            }
+            local transcription_chunk_received_event = {
+                ["idempotencyKey"] = uuid_gen(),
+                ["sessionId"] = transcription["session_id"],
+                ["customerId"] = transcription["customer_id"],
+                ["created"] = util.round(socket.gettime() * 1000),
+                ["meetingFqn"] = transcription["fqn"],
+                ["eventType"] = TRANSCRIPTION_CHUNK_RECEIVED,
+                ["data"] = data
+            }
+            module:log("debug", "Transcription chunk received event: %s", inspect(transcription_chunk_received_event))
```

Comment:
Is it a good idea to do this? Feel like it would burn CPU cycles for little benefit? Maybe comment it out so it's there if / when we need to debug something?

Reply:
Oh I noticed all other event payloads get printed so I tried to follow the trend. They are debug logs, however, and I noticed they don't get printed, does that matter? I guess they still get executed and filtered out. Should I comment all of them?

Reply:
If we do that for all of them let's have it as-is, and if it's a problem let's tackle that as a separate concerted effort.